docs/notebooks/config_to_python.ipynb

```diff
@@ -0,0 +1,508 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "619a87f0-9eff-49c0-a092-7859c2a54d16",
+   "metadata": {},
+   "source": [
+    "The first value returned is a list of classes used by the code snippet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "6ec458e3-1970-42ab-876c-cc7407d9cc19",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['VitessceConfig', 'VitessceConfigDatasetFile']"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e47433f0-0b65-4204-becb-ec9b0c59b731",
+   "metadata": {},
+   "source": [
+    "The second value is the code snippet. When evaluated, the result will be a new `VitessceConfig` instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "6c43ee2a-ac70-4e9b-bcda-a338a8f4871a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "VitessceConfig(\n",
+      "    name=\"Dries\",\n",
+      "    description=\"Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data\",\n",
+      "    schema_version=\"1.0.4\",\n",
+      "    return_self=True,\n",
+      ").add_dataset(\n",
+      "    uid=\"dries-2019\",\n",
+      "    name=\"Dries 2019\",\n",
+      "    files=[\n",
+      "        VitessceConfigDatasetFile(\n",
+      "            url=\"https://s3.amazonaws.com/vitessce-data/0.0.31/master_release/dries/dries.cells.json\",\n",
+      "            data_type=\"cells\",\n",
+      "            file_type=\"cells.json\",\n",
+      "            options=None,\n",
+      "        ),\n",
+      "        VitessceConfigDatasetFile(\n",
+      "            url=\"https://s3.amazonaws.com/vitessce-data/0.0.31/master_release/dries/dries.cell-sets.json\",\n",
+      "            data_type=\"cell-sets\",\n",
+      "            file_type=\"cell-sets.json\",\n",
+      "            options=None,\n",
+      "        ),\n",
+      "    ],\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"embeddingType\", c_scope=\"TSNE\", c_value=\"t-SNE\"\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"embeddingType\", c_scope=\"UMAP\", c_value=\"UMAP\"\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"embeddingZoom\", c_scope=\"TSNE\", c_value=3\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"embeddingZoom\", c_scope=\"UMAP\", c_value=3\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"spatialZoom\", c_scope=\"A\", c_value=-4.4\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"spatialTargetX\", c_scope=\"A\", c_value=3800\n",
+      ").set_coordination_value(\n",
+      "    c_type=\"spatialTargetY\", c_scope=\"A\", c_value=-900\n",
+      ").add_view(\n",
+      "    dataset=\"dries-2019\",\n",
```

Comment:
I think it would be good to clarify with Nils what precisely he wants from the generated node that the json doesn't offer. One thing he might be looking for is avoiding cases like this where the same uuid string is incorporated as multiple points in the code... which might be tricky. Or maybe it's enough to get it into the python syntax and avoid the extra quotes around everything.